src/main/java/edu/hm/hafner/analysis/parser/SimulinkCheckParser.java

```diff
@@ -0,0 +1,131 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.io.File;
+import java.io.IOException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.ReaderFactory;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * A parser for Simulink Check tool by Mathworks results.
+ * Used for .html report files.
+ *
+ * @author Eva Habeeb
+ */
+public class SimulinkCheckParser extends IssuePropertiesParser {
+
+    private static final long serialVersionUID = -8099258658775128275L;
+
+    @Override
+    @SuppressWarnings({"PMD.NcssCount", "PMD.CyclomaticComplexity", "PMD.CognitiveComplexity", "PMD.NPathComplexity" })
```

Comment:
Is this warning related to the code in line 28-53? There is no single `if` contained.

Reply:
Fixed, sorry it was left there from the old version of the code.